WikiWatershed/model-my-watershed
# Feature/lol/one km stamp

Connects #625 

When the user select the one square km drawing tool, they are given a point to
drop on the map. This acts as the center of the square. We use turf
destination, which uses the Haversine formula to calculate four points around
the center at a distance of one half Km with the bearings -180, -90, 0, 90.
This is then used to create a polygon and start the analysis.

To test:

* Run the npm install shell script.
* Bundle with the vendor flag.
* Locate the "1 Km square" button and click it.
* Click anywhere on the map.
* Check to make sure the corners of the box actually are about 1 km apart.
